src/secp256k1.c

```diff
@@ -822,9 +823,10 @@ int secp256k1_xonly_privkey_tweak_add(const secp256k1_context* ctx, unsigned cha
     }
     secp256k1_pubkey_load(ctx, &ge, &pubkey);
     if (!secp256k1_fe_is_quad_var(&ge.y)) {
-        if (!secp256k1_ec_privkey_negate(ctx, seckey32)) {
-            return 0;
-        }
+        secp256k1_scalar_set_b32(&sec, seckey32, NULL);
```

Comment:
Maybe explain why it's okay to ignore the overflow.

Reply:
fixed